exporter/opentelemetry-exporter-datadog/tests/test_datadog_exporter.py

```diff
@@ -151,11 +151,12 @@ def test_translate_to_datadog(self):
             attributes={
                 "key_resource": "some_resource",
                 "service.name": "resource_service_name",
-            }
+            },
+            schema_url="",
         )
 
         resource_without_service = Resource(
-            attributes={"conflicting_key": "conflicting_value"}
+            attributes={"conflicting_key": "conflicting_value"}, schema_url=""
```

Comment:
```suggestion
            attributes={"conflicting_key": "conflicting_value"}, 
            schema_url="",
```

Reply:
Just like in the other repo, this change doesn't pass the lint test.